xml/System.Data.SqlTypes/SqlBoolean.xml

```diff
@@ -996,7 +996,7 @@
         <summary>Computes the bitwise AND operation of two specified <see cref="T:System.Data.SqlTypes.SqlBoolean" /> structures.</summary>
         <returns>The result of the logical AND operation.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.Data.SqlTypes.SqlBoolean.And%2A> oolean)?qualifyHint=True&autoUpgrade=False]]></format>
+          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.Data.SqlTypes.SqlBoolean.And(SqlBoolean, SqlBoolean)>]]></format>
```

Comment:
broken link. uid is System.Data.SqlTypes.SqlBoolean.And(System.Data.SqlTypes.SqlBoolean,System.Data.SqlTypes.SqlBoolean)

and since qualifyHint value was true, we should add ?displayProperty=nameWithType in the end

sentence missing period